services/ui-src/src/utils/other/parsing.ts

```diff
@@ -73,3 +73,11 @@ export const parseFieldLabel = (labelObject: {
 export const parseFieldData = (data: string) => {
   return data || "";
 };
+
+export const parseDynamicFieldData = (data: any) => {
+  const dynamicValues = data?.map((item: any) => item?.name);
+  const formattedValues = dynamicValues
+    ?.map((value: string) => `<p>${value}</p>`)
+    .join(" ");
```

Comment:
Realized after the fact we could probably combine these, no? Use the map in the second one and change value to value.name 🤷

Reply:
Just updated